src/Sulu/Bundle/DocumentManagerBundle/Initializer/WorkspaceInitializer.php

```diff
@@ -39,13 +39,9 @@ public function initialize(OutputInterface $output)
             $workspace = $connection->getWorkspace();
 
             try {
-                $connection->getRootNode();
+                $workspace->createWorkspace($workspace->getName());
                 $output->writeln(sprintf('  [ ] <info>workspace</info>: "%s"', $workspace->getName()));
             } catch (RepositoryException $e) {
-                // TODO: We should catch the more explicit
-                // WorkspaceNotFoundException but Jackalope doctrine-dbal does
-                // not throw this: https://github.com/jackalope/jackalope-doctrine-dbal/issues/322
-                $workspace->createWorkspace($workspace->getName());
                 $output->writeln(sprintf('  [+] <info>workspace</info>: "%s"', $workspace->getName()));
```

Comment:
Should this line really be printed in case of error? If yes I would move it after the try/catch block, because the same line is in the try and catch.